Source/MSBuildTools.Unity/.gitignore

```diff
@@ -58,3 +58,4 @@ sysinfo.txt
 # Crashlytics generated file
 crashlytics-build.properties
 
+/MSBuild/
```

Comment:
This .gitignore comes from GitHub (see line #3). It would be good to be able to update this in the future by just grabbing the latest version from GitHub, and as such, ideally we would not make any manual edits to it. Can this go in a custom .gitignore somewhere else?